diff --git a/services/settings/dumps/gen_last_modified.py b/services/settings/dumps/gen_last_modified.py
index d16eaa01e3..8b6eee590a 100644
--- a/services/settings/dumps/gen_last_modified.py
+++ b/services/settings/dumps/gen_last_modified.py
@@ -55,6 +55,7 @@ def main(output):
         "mobile/ios",
         "comm/mail",
         "comm/suite",
+        "vento",
     ), (
         "Cannot determine location of Remote Settings "
         f"dumps for platform {buildconfig.substs['MOZ_BUILD_APP']}"
@@ -70,6 +71,8 @@ def main(output):
     elif buildconfig.substs["MOZ_BUILD_APP"] == "comm/mail":
         dumps_locations += ["services/settings/dumps/"]
         dumps_locations += ["comm/mail/app/settings/dumps/"]
+    elif buildconfig.substs["MOZ_BUILD_APP"] == "vento":
+        dumps_locations += ["services/settings/dumps/"]
     elif buildconfig.substs["MOZ_BUILD_APP"] == "comm/suite":
         dumps_locations += ["services/settings/dumps/"]
 
